internal/resources/grafana/resource_sso_settings_test.go

```diff
@@ -177,6 +239,18 @@ func testConfigForProvider(provider string, prefix string) string {
 }`, prefix, provider, urls)
 }
 
+const testConfigWithCustomFields = `resource "grafana_sso_settings" "sso_settings" {
+  provider_name = "github"
+  oauth2_settings {
+    client_id     = "client_id"
+    client_secret = "client_secret"
+    custom = {
+      custom_field = "custom1"
```

Comment:
hmm, can users use non camel case here? Someone is going to try it for sure and it should work IMO. Camel Case -> translate, non camel case -> do nothing

Reply:
Good idea, I added a custom field using camelCase. I had to tweak the read function a bit for it to work.